File: problems/1221-element-appearing-more-than-25-in-sorted-array/element-appearing-more-than-25-in-sorted-array.py
class Solution:
    def findSpecialInteger(self, arr: List[int]) -> int:
        hash = {}
        for el in arr:
            if el in hash:
                hash[el] += 1
            else:
                hash[el] = 1
        for key, value in hash.items():
            if value > len(arr) * 0.25:
                return key
        

            

File: problems/1221-element-appearing-more-than-25-in-sorted-array/element-appearing-more-than-25-in-sorted-array.php
class Solution {

    /**
     * @param Integer[] $arr
     * @return Integer
     */
    function findSpecialInteger($arr) {
        $char = [];

        foreach ($arr as $key => $value) {
            $char[$value] = ($char[$value] ?? 0) + 1;
        }

        foreach ($char as $key => $value) {
            if ($value > count($arr) * 0.25) {
                return $key;
            }
        }
    }
}